SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -49,7 +49,11 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
             val apiResult = getAPIResponse<ApiResult>(urlString)
             view.setLoading(false)
             if (apiResult != null) {
-                view.displayTrainTimes(apiResult.toTrainTimes())
+                val trainTimes = apiResult.toTrainTimes()
+                view.displayTrainTimes(trainTimes)
```

Comment:
App would display a blank list if no train times found; we could also just leave the previously fetched train times, but this is fine.

Reply:
That's sort of the point, you don't expect to see any if the app is telling you there are none. I've just left it in there to clear out the previous list.